168_Excel_Sheet_Column_Title.py

```diff
@@ -0,0 +1,13 @@
+# time O(n) n:number of loop
+# space O(m) alphabet list:26, out:26^m
+class Solution:
+    def convertToTitle(self, columnNumber: int) -> str:
+        alphabet = [chr(i) for i in range(65,65+26)]
+        out = ''
+        while columnNumber > 0:
+            remainder = columnNumber%26 - 1
+            out = alphabet[remainder] + out
+            columnNumber = columnNumber//26
```

Comment:
put white spaces before and after the operator

Reply:
insert space